correlation_rules/wiz_issue_followed_by_ssh.yml

```diff
@@ -0,0 +1,50 @@
+AnalysisType: correlation_rule
+RuleID: "Wiz.Issue.Followed.By.SSH"
+DisplayName: "Wiz Issue Followed By SSH to EC2 Instance"
+Enabled: true
+Severity: High
+Description: Wiz detected a security issue with an EC2 instance followed by an SSH connection to the instance. This sequence could indicate a potential security breach.
+Reports:
+  MITRE ATT&CK:
+    - TA0008:T1021.004 # Lateral Movement: Remote Services: SSH
+    - TA0001:T1133 # Initial Access: External Remote Services
+Detection:
+    - Sequence:
+        - ID: WizIssue
+          RuleID: Wiz.Alert.Passthrough
+        - ID: SSH Access
+          RuleID: AWS.VPC.SSHAllowedSignal
+      Transitions:
+        - ID: WizIssue followed by VPC Flow
+          From: WizIssue
+          To: SSH Access
+          WithinTimeFrameMinutes: 60
+          Match:
+            - From: entitySnapshot.externalId
+              To: instanceId
+      LookbackWindowMinutes: 2160
```

Comment:
Can you explain the 12-hour buffer here from the schedule? Is this best practice for CRs?